Vagrantfile

```diff
@@ -13,17 +13,14 @@ Vagrant.configure("2") do |config|
     v.cpus = 2
   end
   config.vm.provision "shell", inline: <<-SHELL
-    curl -OSs https://kojipkgs.fedoraproject.org/packages/criu/3.14/1.fc32/x86_64/criu-3.14-1.fc32.x86_64.rpm
     cat << EOF | dnf -y shell
 config exclude kernel,kernel-core
 config install_weak_deps false
-localinstall criu-3.14-1.fc32.x86_64.rpm
 update
-install iptables gcc make golang-go libseccomp-devel bats jq git-core
```

Comment:
Can we pin the version?

Reply:
> Can we pin the version?

Yes (`criu = 3.14` or something like that) but I don't see the reason why. If the next criu release fails, we want to know about it (and can certainly use a previous release once we do)